src/app/betting/feature/place-bet-page.component.ts

```diff
@@ -66,13 +77,111 @@ import { BetStatus } from '../../shared/util/supabase-helpers';
       />
     </label>
 
-    <!-- Bet terms -->
+    <!-- Bet Type Dropdown -->
+    <!-- eslint-disable-next-line -->
     <label class="flex flex-column gap-2 mt-5">
-      Bet Terms
-      <textarea rows="2" pInputTextarea [(ngModel)]="terms" [required]="true">
-      </textarea>
+      Bet Type
+      <p-dropdown
+        [options]="betTypes"
+        [(ngModel)]="selectedBetType"
+        optionLabel="betTypeString"
+        optionValue="betType"
+        styleClass="flex"
+      />
     </label>
 
+    @if (selectedBetType() === BetType.DuelWinner) {
```

Comment:
I would change this `@if / @else if / @else` to a `@switch` statement. This will help especially once we add more bet types.

```html
@switch (selectedBetType) {
  @case (BetType.DuelWinner) {
    <!-- ... -->
  }
  @case (BetType.SpecialSpaceEvent) {
    <!-- ... -->
  }
  @case (BetType.Manual) {
    <!-- ... -->
  }
  @default {
    Unknown bet type: {{ selectedBetType() }}
  }
}
```